database/migrations/2019_04_30_183052_workshop_attendence.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
Seems to be an empty migration. This doesn't spark joy

Reply:
oh oops didnt mean to commit dat boi